ci/rpm/install-base-el8.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -e
+
+# Install OS updates and packages as required for building DAOS on EL 8 and
+# derivatives.  Include basic tools and daos dependencies that come from the core repos.
+# This is for a daos test install environment, not a build environment.
+
+# This script use used by docker but can be invoked from elsewhere, in order to run it
+# interactively then these two commands can be used to set dnf into automatic mode.
+# dnf --assumeyes install dnf-plugins-core
+# dnf config-manager --save --setopt=assumeyes=True
+# CI environment expects git command to be present.
+
+dnf --nodocs install \
+    bzip2 \
+    diffutils \
+    e2fsprogs \
+    file \
+    flex \
+    fuse3 \
+    git \
+    glibc-langpack-en \
+    golang \
+    ipmctl \
+    java-1.8.0-openjdk \
+    Lmod \
+    ndctl \
+    numactl \
+    sg3_utils \
+    sudo \
+    systemd \
+    which
```

Comment:
Why doesn't installing the DAOS RPMs bring any/all of this in?  Is **all** of this needed just to install DAOS RPMs?  Much of this looks like packages that would be installed in the base image or is a **build** requirement, not run-time requirement.

Reply:
Installing the DAOS images will bring them in.   This is just pre-installing them in the cache layer to speed things up

Reply:
I'm not sure how I feel about that, on the one hand then it's good to keep the server load down and docker image re-use up, but on the other hand a local pull isn't that expensive and there's a cost to keeping this up-to-date.

Reply:
> I'm not sure how I feel about that, on the one hand then it's good to keep the server load down and docker image re-use up, but on the other hand a local pull isn't that expensive and there's a cost to keeping this up-to-date.

And a risk of papering over dependency specifications in the packages.